django/contrib/staticfiles/apps.py

```diff
@@ -5,3 +5,4 @@
 class StaticFilesConfig(AppConfig):
     name = 'django.contrib.staticfiles'
     verbose_name = _("Static Files")
+    collectstatic_ignore_patterns = ['CVS', '.*', '*~']
```

Comment:
+1 on just `ignore_patterns` since collectstatic isn't the only endpoint where this may come in handy (e.g. django-pipeline uses finders, too).